### ISSUE TITLE
Connect CourtListener application code to Elastic Search

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#2958]
In #2950 we got the Elastic Cluster connected via a load balancer to the CL application code, network-wise. Now we need to actually do it from the application. This is a bit tricky, because of encryption and certificates.

Relates to: #2867.

**Comment by mlissner** [comment]
It turns out this is pretty hard because the application code connects to the internal AWS ALB, and it forwards traffic to the pods. The pods have self-signed certs, but by the time the application gets those certs, they've been chained to the ALB's certificate. @albertisfu can chime in if I didn't get that right, but that's my understanding. 

What we tried to do was get the cert:

```
kubectl get secret elastic-cluster-es-http-certs-public -o=jsonpath='{.data.tls\.crt}' | base64 --decode > ca.crt
```

And then connect using it, with something like:

```
curl https://xxx.us-west-2.elb.amazonaws.com:9200 -u elastic:xxx --cert ca.crt
```

That threw an error:

```
curl: (58) unable to set private key file: 'ca.crt2' type PEM
```

So we played with the `--key` flag instead of `--cert`, which maybe helped, but ultimately, the easiest solution is to just not verify the cert, since this is inside our cluster anyway, so we do that in https://github.com/freelawproject/courtlistener/pull/2960
